client/task-list/tasks/payments/index.js

```diff
@@ -42,36 +42,34 @@ class Payments extends Component {
 			( method ) => ( enabledMethods[ method.key ] = method.isEnabled )
 		);
 		this.state = {
+			busyMethod: null,
```

Comment:
Nice simplification, only allowing a single "busy" method makes the code much simpler, and I don't think many users will be complaining that they can't install several methods at once. I definitely overthought it :)

Reply:
I actually liked the way you implemented it and the idea of being able to quickly install multiple gateways.  The main reason I ended up shifting back towards a single is since multiple install/connection flows may stir up some bugs.  But its definitely something to consider in the long-run.